frontend/src/component/feature/FeatureStrategy/FeatureStrategyConstraints/FeatureStrategyConstraintAccordionList/FeatureStrategyConstraintAccordionList.tsx

```diff
@@ -113,14 +132,34 @@ export const FeatureStrategyConstraintAccordionList = forwardRef<
                                 constraints={constraints}
                                 state={state}
                             />
+
+                            <Box
+                                sx={(theme) => ({
+                                    marginTop: theme.spacing(2),
+                                    marginBottom: theme.spacing(2),
+                                })}
```

Comment:
Does this get around the problem of the extra margin somehow? If the box is empty, won't this render at least `theme.spacing(4)` of margin (bottom + top)? And if not, why not?

Reply:
could it be margin collapsing between top and bottom when the content is empty?

<img width="786" alt="Screenshot 2024-07-08 at 13 06 36" src="https://github.com/Unleash/unleash/assets/1394682/b4ccc08f-b043-4d1e-994c-97f040316a3e">